db/README.md

```diff
@@ -23,7 +23,7 @@ _Please keep this up to date as the schema changes!_
   * `druid` is the digital resource unique identifier
   * `current_version` is current latest version we've seen for the druid across all instances.
   * `preservation_policy_id` points to the policy governing how the object should be preserved.
-* A `CompleteMoab` represents a physical copy of a `PreservedObject`, e.g. a moab (represents all versions) stored on premesis and accessed via NFS mount, the `zipped_moab_vesion` (represents a single version) copy sitting on a cloud endpoint (manipulable via REST calls to a 3rd party cloud service), etc.
+* A `CompleteMoab` represents a physical copy of a `PreservedObject`, e.g. a moab (represents all versions) stored on premesis and accessed via NFS mount, the `zipped_moab_vesion` (represents a single version) copy sitting on a cloud endpoint (manipulable via REST calls to a 3rd party cloud service), etc. Multiple `CompleteMoab` instances can share the same `PreservedObject` instance. 
```

Comment:
Are you open to using `belong to` instead of `share`? That makes the relationship a tad clearer and aligns with the language of our tooling (ActiveRecord).